src/pages/Follow/components/FollowList.module.scss

```
@@ -26,4 +28,25 @@
       transform: rotate(180deg);
     }
   }
+
+  @include media.media-breakpoint-down(mobile) {
+    width: 85vw;
+  }
+}
+
+.recent {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  overflow: scroll;
+  gap: 10px;
+  width: 1067px;
+
```

인터넷 익스플로러와 파이어폭스도 추가하면 어떨까요?
```suggestion
  -ms-overflow-style: none; /* 인터넷 익스플로러 */
  scrollbar-width: none; /* 파이어폭스 */
```

---

IE대응은 필요없을것같은데 파폭은 좋아보여요

---

파이어폭스 추가 완료했습니다